src/cascade/input_data/configuration/form.py

```diff
@@ -49,9 +50,8 @@ def _full_form_validation(self, root):
             lower = self.min
             upper = self.max
             mean = self.mean
-            if mean is None:
-                if np.isinf(lower) or np.isinf(upper):
-                    mean = max(lower, 0)
+            if mean is None and np.isinf(lower) or np.isinf(upper):
```

Comment:
Do we need parens here?  if mean is None and (np.isinf(lower) or np.isinf(upper))

Say, mean is 3, and upper is np.inf and lower is 1, then mean will be set to 1.  
Can lower and upper be None?  doesn't look like np.isinf() will like it, if so?